cmd/argocd/commands/app.go

```diff
@@ -381,6 +396,19 @@ func NewApplicationGetCommand(clientOpts *argocdclient.ClientOptions) *cobra.Com
 			})
 			errors.CheckError(err)
 
+			if sourceName != "" && sourcePosition != -1 {
```

Comment:
Since the `Name` parameter has been added to the `ApplicationSource` object, it's not exclusive to multi-source applications. This means the `name` field also applies to single-source apps. Therefore, in all commands where we’re adding the `sourceName` check, we should ensure the command supports single-source applications as well, using the `sourceName` field.

Reply:
That works for me. Although, let's update the comment on the `Name` field in types.go to reflect that it is mainly used for multi-source apps.